### PR TITLE
 Roc 2703 save part admission response

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-2703

contains https://github.com/hmcts/cmc-citizen-frontend/pull/644 changes as well.

### CODE REVIEW

File: src/main/app/claims/models/response/partDefenceAdmission.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  defence: string,
+  timeline: DefendantTimeline
+  evidence: DefendantEvidence
+  paymentOption?: PaymentOption
+  paymentDate?: Moment
+  repaymentPlan?: RepaymentPlan
+  statementOfMeans?: StatementOfMeans
+}
+
+export namespace PartAdmissionResponse {
```
Comment (yarpo) [inline src/main/app/claims/models/response/partDefenceAdmission.ts]: as we have `PartialAdmission` as a draft why don't we have `PartAdmissionResponse`?

File: config/default.json [hunk 2]
```diff
@@ -84,7 +84,7 @@
     "testingSupport": false,
     "countyCourtJudgment": false,
     "fullAdmission": false,
-    "partialAdmission": false,
+    "partialAdmission": true,
```
Comment (timja-kainos) [inline config/default.json]: this is on by default locally revert change please

File: src/main/features/response/draft/responseDraft.ts [hunk 3]
```diff
@@ -140,11 +152,18 @@ export class ResponseDraft extends DraftDocument {
   // is incomplete. ROC-3658 should revisit once 'statement of means' flow is complete.
```
Comment (sabahirfan) [inline src/main/features/response/draft/responseDraft.ts]: I think we can remove this comment now.

File: src/main/app/paths.ts [hunk 4]
```diff
@@ -14,6 +14,7 @@ export class Paths {
   // Ajax endpoints
   static readonly postcodeLookupProxy = new RoutablePath('/postcode-lookup', false)
   static readonly paymentPlanCalculation = new RoutablePath('/payment-plan-calculation', false)
+  static readonly totalIncomeOrExpensesCalculation = new RoutablePath('/total-income-expense-calculation',false)
```
Comment (yarpo) [inline src/main/app/paths.ts]: space afer `,`

**Pull request closed without merging**